### PR TITLE
v1 of data model

### PR DESCRIPTION
**Pull request by james-labyer**

### CODE REVIEW

File: docs/db.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Data Model
+
+## Tables
+
+### Users
+
+| Property | Type    | Notes                                                          |
+| -------- | ------- | -------------------------------------------------------------- |
+| id       | Integer | Auto-generated Primary Key for this row                        |
```
Comment (carlosmmc) [inline docs/db.md]: nit - should we use uuid's for this project, if so let's specify it here.

File: docs/db.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+| api_id   | String  | 26-character public-facing id made up of numbers and letters   |
+| email    | String  | Email address used to sign up for the service                  |
```
Comment (carlosmmc) [inline docs/db.md]: nit - for types let's use Postgres types so for String let's declare how many characters we plan to use for the varchar.

File: docs/db.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+| Property    | Type    | Notes                                                      |
+| ----------- | ------- | ---------------------------------------------------------- |
+| id          | Integer | Auto-generated Primary Key for this row                    |
+| uid         | Integer | User that owns this key. Foreign Key on Users.             |
+| api_id      | String  | 26-character public-facing id made up of numbers & letters |
+| public_key  | String  | JWK with the unencrypted public key                        |
+| private_key | String  | Encrypted private key                                      |
+| account_key | String  | Key that unlocks the user's data                           |
+
+## Relationships Between Entities
```
Comment (carlosmmc) [inline docs/db.md:35]: nit - should we also define in this doc what indexes we plan to define?

**Review by carlosmmc — COMMENTED** [review]
looking good! I left some nits, no worries if you don't plan to incorporate them I can approve as is if you'd like

File: docs/db.md (hunk 3 above)
Comment (james-labyer) [reply docs/db.md:35]: @carlosmmc PostgreSQL creates indexes on unique columns [automatically](https://www.postgresql.org/docs/current/indexes-unique.html), and that should be all we need.

**Review by carlosmmc — APPROVED** [review]
looks awesome!

**Pull request merged**